libs/wingii/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@winglang/wingii",
   "private": true,
   "scripts": {
-    "compile": "cargo wasi build --release",
+    "compile": "cargo build --target wasm32-wasi --release",
```

Comment:
Is there a 32-bit WASM and 64-bit WASM? 😱

Reply:
Only 32bit is really supported for now. As with most of wasm, 64bit is on its way and will probably take a while.